drms/utils.py

```diff
@@ -73,11 +67,92 @@ def to_datetime(tstr, force=False):
     result : pandas.Series or pandas.Timestamp
         Pandas series or a single Timestamp object.
     """
-    s = pd.Series(tstr).astype(str)
+    s = pd.Series(tstr, dtype=object).astype(str)
     if force or s.str.endswith('_TAI').any():
-        s = s.str.replace('_TAI', '')
+        s = s.str.replace('_TAI', "")
         s = s.str.replace('_', ' ')
         s = s.str.replace('.', '-', n=2)
     res = _pd_to_datetime_coerce(s)
-    res = res.dt.tz_localize(None)  # remove any timezone information
+    res = res.dt.tz_localize(None)
     return res.iloc[0] if (len(res) == 1) and np.isscalar(tstr) else res
+
+
+def generate_changelog_for_docs(directory, output_filename=None):
```

Comment:
This is also in the sphinx extension in core

Reply:
I am not adding a sunpy dep to make change logs.